lib/active_shipping/rate_estimate.rb

```diff
@@ -66,13 +66,19 @@ module ActiveShipping
   # @!attribute delivery_category
   #   The general classification of the delivery method
   #   @return [String]
+  #
+  # @!attribute shipment_options
+  #   Additional priced options bundled with the given rate estimate with price in cents
+  #   @return [Array<{ code: String, price: Integer }>]
```

Comment:
The most optimal case is to use a `ShipmentOption` class https://github.com/Shopify/active_shipping/pull/387 but the interface is still ambiguous at this point.

An inline object will allow us to iterate on the interface quickly until we're confident with what attributes belongs in `ShipmentOption`.